app/redux/selectors/SetsSelectors.js

```diff
@@ -367,3 +367,37 @@ export const getSlowestDurationEver = (state, set) => {
 };
 
 export const getRevision = (state) => stateRoot(state).revision;
+
+// 1rm
+
+export const getExerciseVelocities = (state, exercise) => {   
+    const historySets = getHistorySets(state);
+    const workoutSets = getWorkoutSets(state);
+
+    const sets = historySets.concat(workoutSets);
```

Comment:
- [ ] Make a getAllSets getter and use it (as apparently we're doing it over and over)